homeworks/denys-yorzh/homework-1/second.js

```diff
@@ -0,0 +1,4 @@
+const firstNumb = promt ("Enter the first number","")
```

Comment:
Incorrect code, have you tested it?
```suggestion
const firstNumb = prompt("Enter the first number")
```